File: IOT/gurpreetk12_gurpreetkaur_2125cs1094_2/FunctionandArrays/AnyBaseToDecimal.cpp
#include<iostream>
#include<math.h>
using namespace std;
int AnyToDec(int n, int b) {
    int g=n;
    int p=0,i=0,sum=0;
    int q=g;
    while(g!=0)
    {
        q=g%10;
        p=pow(b,i);
        
        i++;
        g=g/10;
        sum=sum+(p*q);
    }
    return(sum);
    
}
int main() {
  int n;
  int b;
  int sum=0,h=0,g=0,i=0;
  cin >> n;
  cin >> b;
  int res = AnyToDec(n, b);
  cout << res << endl;
}

File: AR-VR/kapilsingh2003_KapilKumarSingh_2125cs1169_2/FunctionAndArrays/AnyBaseToDecimal.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      Scanner scn = new Scanner(System.in);
      int n = scn.nextInt();
      int b = scn.nextInt();
      int d = getValueIndecimal(n, b);
      System.out.println(d);
   }
  
   public static int getValueIndecimal(int n, int b){
      int i=0, sum=0;
        while(n>0){
            int x = n%10;
            sum = sum + (x *(int)Math.pow(b,i++));
            n = n/10;
        }
        return sum;
   }
  }